SpatialGDK/Documentation/features.md

```diff
@@ -48,6 +48,10 @@ Actor property handover is a new feature we’re introducing with the SpatialOS
 
 `Singleton Actors` is a new term that describes Actors which contain global state and logic.  Only one of these Actors exists in a game world. The SpatialOS Unreal GDK supports both client accessible and server only singletons (respectively known as public and private singletons). For more information see the documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors).
 
+## Cross-server RPCs
+
+`Cross-server RPCs` is a new type of RPC that functions between two servers. These will enable a server without authority over an Actor to communicate with the server which has authority and is able to manipulate the Actor. For more information, see [Cross-server RPC]({{urlRoot}}/content/dummy-link) documentation.
```

Comment:
Change the first sentence into the plural, i.e. "`Cross-server RPCs` are a new type of RPC that function between two servers."

These will enable -> These enable

Also add "the" before [Cross-server RPC]({{urlRoot}}/content/dummy-link)